runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1445,10 +1445,12 @@ gcParseXXArguments(J9JavaVM *vm)
 				goto _error;
 			}
 
-			if (0 == extensions->checkpointGCthreadCount) {
+			if (0 == extensions->userSpecifiedParameters._checkpointGCThreads._valueSpecified) {
 				j9nls_printf(PORTLIB, J9NLS_ERROR, J9NLS_GC_OPTIONS_VALUE_MUST_BE_ABOVE, "-XX:CheckpointGCThreads=", (UDATA)0);
 				goto _error;
 			}
+			extensions->userSpecifiedParameters._checkpointGCThreads._wasSpecified = true;
+			extensions->checkpointGCthreadCount = extensions->userSpecifiedParameters._checkpointGCThreads._valueSpecified;
```

Comment:
can't we just get rid of extensions->checkpointGCthreadCount?

Reply:
I think we definitely could, but we already did the same for parameters like extensions->memoryMax, extensions->newSpaceSize, etc. Do we want to use the userSpecifiedParameters just for this one?

Reply:
we are inconsistent (there is an example of dnssExpectedRatioMinimum where we don't both variables), but there is probably a reason for that....

if we do expect to further adjust it after parsing, it probably makes sense to keep it both (so we know the originally specified value for debuging reasons and what not).

memoryMax is indeed adjusted (page alignment etc), and if we indeed need to adjust checkpointGCthreadCount, we can keep ext-> checkpointGCthreadCount

Reply:
We do adjust it in the MM_ConfigurationDelegate::checkPointGCThreadCountVerifyAndAdjust() though:
---
		if (!extensions->userSpecifiedParameters._checkpointGCThreads._wasSpecified) {
			extensions->checkpointGCthreadCount = OMR_MIN(extensions->checkpointGCthreadCount, extensions->gcThreadCount);
		} else if (
---